src/alert.cpp

```diff
@@ -245,8 +247,22 @@ bool CAlert::ProcessAlert(const std::vector<unsigned char>& alertKey, bool fThre
     return true;
 }
 
-void
-CAlert::Notify(const std::string& strMessage, bool fThread)
+// static
+void CAlert::NotifyInternal(const std::string& strMessage)
+{
+    static CAlert internalAlert;
+    internalAlert.vchMsg = std::vector<unsigned char>(strMessage.begin(), strMessage.end());
```

Comment:
IMO you also should set `internalAlert.strStatusBar`. (vchMsg is the raw data of a network alert that comes with a signature).